src/gluonts/nursery/duru/data.py

```diff
@@ -0,0 +1,488 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+import gluonts.dataset
+import torch
+import torch as T
+import numpy as np
+import sklearn.model_selection
+import pickle
+import os
+import torchvision.transforms as transforms
+from gluonts.dataset.repository.datasets import get_dataset, dataset_recipes
+from gluonts.dataset.split import OffsetSplitter
+from utils import calculate_padding_base_2
+from gluonts.dataset.rolling_dataset import StepStrategy, NumSplitsStrategy
+from gluonts.dataset.rolling_dataset import generate_rolling_dataset
+
+
+def set_up_data(H):
```

Comment:
What's `H`? I generally have an aversion against single-letter variable names.

Reply:
H is the parsed command line object which contains all hyperparameters. I find the single-letter variable name convenient here since it's abundantly used and hence it should be clear what it means.

Reply:
It sure sounds convenient, but for someone new to the code it is not obvious. Code is read more often then written, so I will insist on changing this to something more self explainatory.